remoter-server/test.json

```diff
@@ -0,0 +1,11 @@
+{
+    "experimentId": "e52b39624588791a7889e39c617f669e",
+    "taskName": "test_fn",
+    "config": {
+        "enabled": true,
+        "filterSettings": {
+        "minCellSize": 10800,
+        "binStep": 200
+        }
+    }
+}
```

Comment:
A sample task input that we might get in a pipeline.

Reply:
to make it obvious that this is a sample task input, you could rename the file from `test.json` to `sample_task_input.json`

Reply:
Done